docs/guides/dev_guide.rst

```diff
@@ -249,6 +249,16 @@ We also do a few things that aren't explicitly called out in PEP 8:
           CertainQuestionsError: Raises certain questions about the authors sanity.
         """
 
+* Use `TODO` comment for workaround, temporary or not perfect code.
+  A `TODO` comment begins with the string `TODO` all in caps. It can use with some identifier like
+  Name or Email (responsible for improvement), Bugzilla, Github Issue etc.
+  An example is described below::
```

Comment:
Fix typo with `::`, change to `:`

Reply:
Optional: this can just read `For example:`

Reply:
It's not typo.